README.md

```diff
@@ -38,41 +43,99 @@ on to create a full desktop environment.
 Labwc tries to stay in keeping with [wlroots] and [sway] in terms of general
 approach and coding style.
 
-Labwc only understands [wayland-protocols] &amp; [wlr-protocols], and it cannot
-be controlled with dbus, sway/i3-IPC or other technology. The reason for this is
-that we believe that custom IPCs and protocols create a fragmentation that
-hinders general Wayland adoption.
+Labwc has no reliance on any particular Desktop Environment, Desktop Shell or
+session. Nor does it depend on any UI toolkits such as Qt or GTK.
+
+### 1.1 Why?
+
+We believe that there is a need for a simple Wayland compositor which strikes a
+balance between minimalism and bloat approximately at the level where openbox
+resides in the X11 domain. This does not mean that it should look square and
+pixelated like something out of the 1990s or indeed that the improved
+performance offered by Wayland should be compromised.
+
+It is important to us that scope is tightly controlled so that the compositor
+matures into production quality. On the whole, we value robustness, reliability
+stability and simplicity over new features. This is because we believe that
+coming up with and implementing new ideas is easy, but making them stick and
+work for a long time hard. We want the latter.
```

Comment:
"long time is hard" (missing word)

Reply:
~hard~ difficult

Reply:
~This is because we believe that
coming up with and implementing new ideas is easy, but making them stick and
work for a long time hard. We want the latter.~

Coming up with new ideas and features is easy. Maintaining and stabilizing them is not. 

_Just my 2 cents to keep it simple._